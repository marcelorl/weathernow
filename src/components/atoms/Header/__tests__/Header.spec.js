import createTestContext from 'react-cosmos-test/enzyme'

import fixture from '../Header.fixture'

const { mount, getWrapper } = createTestContext({fixture})

beforeEach(mount)

describe('#Header atom', () =>
  it('renders component', () => {
    expect(getWrapper()).toMatchSnapshot()
  })
)
