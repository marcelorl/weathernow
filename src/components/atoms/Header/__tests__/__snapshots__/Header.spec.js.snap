// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Header atom renders component 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #ffffff;
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 3rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0.7rem 0;
  position: fixed;
  top: 0;
  width: 100%;
}

.c0 img {
  height: 100%;
}

<Header
  title="Layer"
>
  <styled.div>
    <StyledComponent
      forwardedComponent={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "lastClassName": "c0",
            "rules": Array [
              "
  align-items: center;
  background: #ffffff;
  box-sizing: border-box;
  display: flex;
  height: 3rem;
  justify-content: center;
  padding: 0.7rem 0;
  position: fixed;
  top: 0;
  width: 100%
  
  img {
    height: 100%;
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
    >
      <div
        className="c0"
      >
        <img
          alt="logo"
          className="weathernow-logo"
          src="logo.svg"
        />
      </div>
    </StyledComponent>
  </styled.div>
</Header>
`;
